lib/spaceship/tunes/app_analytics.rb

```diff
@@ -0,0 +1,145 @@
+module Spaceship
+  module Tunes
+    class AppAnalytics < TunesBase
+      # @return (Integer) The app ID to grab analytics for
+      attr_accessor :apple_id
+
+      attr_mapping({
+        'adamId' => :apple_id
```

Comment:
No clue, I was just following the structure of the other classes. Since the Apple ID is already exposed in application it probably doesn't make sense here since if we're calling this we already have an application